classes/PodsAPI.php

```diff
@@ -1203,6 +1203,18 @@ public function get_wp_object_fields( $object = 'post_type', $pod = null, $refre
 					'type'  => 'slug',
 					'alias' => array( 'nicename', 'slug', 'permalink' )
 				),
+				'first_name'  => array(
+					'name'  => 'first_name',
+					'label' => 'First Name',
+					'type'  => 'text',
+					'alias' => array( 'firstname' )
+				),
+				'last_name'  => array(
+					'name'  => 'last_name',
+					'label' => 'Last Name',
+					'type'  => 'text',
+					'alias' => array( 'lastname' )
```

Comment:
* :no_entry_sign: Each line in an array declaration must end in a comma
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: fac1eb86362064fe90d1d11060a16f35</sub>